openstack/sharedfilesystems/v2/securityservices/requests.go

```diff
@@ -0,0 +1,52 @@
+package securityservices
+
+import "github.com/gophercloud/gophercloud"
+
+// CreateOptsBuilder allows extensions to add additional parameters to the
+// Create request.
+type CreateOptsBuilder interface {
+	ToSecurityServiceCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts contains options for creating a SecurityService. This object is
+// passed to the securityservices.Create function. For more information about
+// these parameters, see the SecurityService object.
+type CreateOpts struct {
+	// The security service name
+	Name string `json:"name"`
```

Comment:
Can the name by empty or should this have an `omitempty` or `required:true` tag?